storage/sas_token.go

```diff
@@ -82,10 +82,15 @@ func ParseAccountSASConnectionString(connString string) (map[string]string, erro
 	for _, atoken := range tokens {
 		// The individual k-v are separated by an equals sign.
 		kv := strings.SplitN(atoken, "=", 2)
+		if len(kv) != 2 {
+			return nil, fmt.Errorf("[ERROR] token `%s` is an invlaid key=pair (connection string %s)", atoken, connString)
```

Comment:
minor: typo in 'invlaid'

Reply:
thanks for spotting this 🙂 

```suggestion
			return nil, fmt.Errorf("[ERROR] token `%s` is an invalid key=pair (connection string %s)", atoken, connString)
```